### DISCUSSION TITLE
[Project template] Add project template metadata to be used with mustache.js

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#247] [Ideas]
### Which area of this project could be most improved?

Marketplace

### Description

Hi guys,
reading the public documentation you talk about the usage of _mustache.js_ during the project creation:
https://docs.mia-platform.eu/docs/development_suite/company/project-templates#how-to-create-a-project-archive

At the moment _mustache.js_ gets, during the project creation phase, the following arguments:
- projectId;
- name;
- configurationGitPath;
- environments.

Could you add a `metadata` field to be configured on the project template that can be used with _mustache.js_?
This should be very useful.

Thanks.

**Comment by davidebianchi** [comment]
Hi @silversoul93! Can you describe your use case?

**Reply by frengooo** [reply]
Hi @davidebianchi, a client needs to create many specific project templates to release its products.
To create specific project templates the client asks us to have a `metadata` field to add custom fields to be used with _mustache_ during the project creation.

I think this addition could improve the project template customization level.

**Reply by frengooo** [reply]
Hi, any news on this discussion?

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
